ctor/src/lib.rs

```diff
@@ -134,24 +135,30 @@ pub fn ctor(_attribute: TokenStream, function: TokenStream) -> TokenStream {
         // Why .CRT$XCU on Windows? https://www.cnblogs.com/sunkang/archive/2011/05/24/2055635.html
         // 'I'=C init, 'C'=C++ init, 'P'=Pre-terminators and 'T'=Terminators
 
+        let ctor_ident =
+            syn::parse_str::<syn::Ident>(format!("{}___rust_ctor___ctor", ident).as_ref())
+                .expect("Unable to create identifier");
+
         let output = quote!(
             #[cfg(not(any(target_os = "linux", target_os = "android", target_os = "freebsd", target_os = "macos", target_os = "ios", windows)))]
             compile_error!("#[ctor] is not supported on the current target");
 
+            #(#attrs)*
+            #vis #unsafety extern #abi #constness fn #ident() #block
```

Comment:
Since this is no longer exported, does it still need `extern`?

Reply:
The `extern` is needed to support non-Rust ABIs for the underlying ctor function, eg
```
#[ctor]
extern "stdcall" fn foo() {}
```

I figured it made sense to keep support, as just `extern` is equivalent to no `extern` at all (when used on a function not in an `extern` block, according to the Rust reference)

Reply:
In the end, I'm neutral on it. If you think it's not a significant case to support, I'm fine with dropping it

Reply:
That makes sense to me. Let's keep it.